aurora/bag_transfer/mixins/authmixins.py

```diff
@@ -7,7 +7,7 @@
 
 
 class LoggedInMixinDefaults(LoginRequiredMixin):
-    login_url = "/app"
+    login_url = "/login"
```

Comment:
I am wondering if we can use `reverse_lazy` here to avoid hardcoding a URL?

```suggestion
    login_url = reverse_lazy("login")
```